Sources/ScipioKit/DescriptionPackage.swift

```diff
@@ -112,9 +112,9 @@ struct DescriptionPackage: PackageLocator {
 }
 
 extension DescriptionPackage {
-    func resolveBuildProducts() throws -> OrderedSet<BuildProduct> {
```

Comment:
We should be able to remove `import Collections` in this file.

Reply:
I removed the import, thank you!